doc/source/reference/distutils_status_migration.rst

```diff
@@ -0,0 +1,88 @@
+.. _distutils-status-migration:
+
+Status of ``numpy.distutils`` and migration advice
+==================================================
+
+`numpy.distutils` has been deprecated in NumPy ``1.23.0``. It will be removed
+for Python 3.12; for Python <= 3.11 it will not be removed until 2 years after
+the Python 3.12 release (Oct 2025).
+
+
+Migration advice
+----------------
+
+It is **not necessary** to migrate immediately - the release date for Python 3.12
+is October 2023, so you have a good amount of time left. It may be beneficial
+to wait with migrating until there are examples from other projects to follow
+(see below).
+
+There are several build systems which are good options to migrate to. Assuming
+you have compiled code in your package (if not, use Flit_) and you want to be
+using a well-designed, modern and reliable build system, your two best options
```

Comment:
I'll try a small rephrase. I don't think it's a controversial opinion though - not sure what the third-best build system would be, but it will be a pretty distant third either way.

Reply:
Maybe a tiny rephrase, since there are other good ones as well, and Flit doesn't support SCM versioning, but it's 0-dependency, really simple (in the backend code), and part of the pypa, so hard not to recommend. Though @takluyver might not like a recommendation here, I don't know, he seems to not like "official" recommendations to use Flit. ;)

Reply:
I'll change to "we recommend". I think Thomas will be fine with a recommendation. Either way, we need to recommend something. This is aimed at people who don't know what they want/need, and we're already giving them four options - which is quite a bit.

Reply:
I'm OK with NumPy recommending Flit - my concern was about recommendations in 'official' PyPA documentation that might be seen as making it the standard tool.

Reply:
Thanks, adopted the "we recommend" rephrase, and kept the Flit recommendation.